src/sass/pages/cashier.scss

```diff
@@ -105,7 +105,15 @@
     width: 111px;
     height: 38px;
 }
-
+.button_cashier {
```

Comment:
do we really need to add custom styling to the button? I would be better to keep the number of CSS tweaks down to a minimum. Ideally the entire site uses only what is listed on https://www.binary.com/styles without any further tweaks anywhere.